### PR TITLE
Task 1

### PR DESCRIPTION
**Pull request by Nishikantkp**

### CODE REVIEW

File: isOlder.cpp [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+ERROR CASES: Return -1 if any DOB is invalid.
+
+NOTES: 		Don't use any built-in C functions for comparisions. You are free to write any helper functions.
+*/
+int isOlder(char *dob1, char *dob2) {
+	{
+		int day1, day2, mon1, mon2, yr1, yr2;
+		int day_limit1 = 32, day_limit2 = 32;
+		sscanf(dob1, "%d-%d-%d", &day1, &mon1, &yr1);
```
Comment (Abhivardhan) [inline isOlder.cpp]: you cant scan the input, the input will be given to you as a string. please do the problem again

File: isPerfect.cpp [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+				if (index*index != num)
+					sum = sum + index1 + num / index1;
+				else
+					sum = sum + index;
+			}
+		}
+
+		if (sum == num)
+		{
+
```
Comment (Abhivardhan) [inline isPerfect.cpp]: remove this extra line

File: nthNumber.cpp [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+			{
+				return num1;
+			}
+			num3 = num1 + num2;
+			num1 = num2;
+			num2 = num3;
+			count++;
+		}
+	}
+	else
```
Comment (Abhivardhan) [inline nthNumber.cpp]: error check must be at the start

File: isOlder.cpp (hunk 1 above)
Comment (Nishikantkp) [reply isOlder.cpp]: There I am  assigning values from string to variables by converting them into integer types

Comment (Abhivardhan) [reply isOlder.cpp]: write the logic for the same instead of sscanf

Comment (Nishikantkp) [reply isOlder.cpp]: can i use atoi() function?

**Pull request closed without merging**